Vagrantfile

```diff
@@ -0,0 +1,17 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
```

Comment:
Not sure if this file helps anyone else. I wanted it as my main dev host is Windows.
Let me know if I should remove it.

Reply:
this is ok